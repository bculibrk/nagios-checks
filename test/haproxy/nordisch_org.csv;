# pxname,svname,qcur,qmax,scur,smax,slim,stot,bin,bout,dreq,dresp,ereq,econ,eresp,wretr,wredis,status,weight,act,bck,chkfail,chkdown,lastchg,downtime,qlimit,pid,iid,sid,throttle,lbtot,tracked,type,rate,rate_lim,rate_max,check_status,check_code,check_duration,hrsp_1xx,hrsp_2xx,hrsp_3xx,hrsp_4xx,hrsp_5xx,hrsp_other,hanafail,req_rate,req_rate_max,req_tot,cli_abrt,srv_abrt,comp_in,comp_out,comp_byp,comp_rsp,lastsess,last_chk,last_agt,qtime,ctime,rtime,ttime,agent_status,agent_code,agent_duration,check_desc,agent_desc,check_rise,check_fall,check_health,agent_rise,agent_fall,agent_health,addr,cookie,mode,algo,conn_rate,conn_rate_max,conn_tot,intercepted,dcon,dses,
http,FRONTEND,,,2,48,10000,39006,1115049190,267430337,0,0,386,,,,,OPEN,,,,,,,,,1,3,0,,,,0,1,0,62,,,,0,34879,4791,2098,39,14,,1,62,41822,,,0,0,0,0,,,,,,,,,,,,,,,,,,,,,http,,1,62,39006,0,0,0,
http,sock-1,,,0,7,10000,2859,916485,450245,0,0,367,,,,,OPEN,,,,,,,,,1,3,1,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http,,,,,,0,0,
http,sock-2,,,0,1,10000,52,12785,10016,0,0,19,,,,,OPEN,,,,,,,,,1,3,2,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http,,,,,,0,0,
http,sock-3,,,2,48,10000,36095,1114119920,266970076,0,0,0,,,,,OPEN,,,,,,,,,1,3,3,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http,,,,,,0,0,
radicale,radicale-local,0,0,0,1,,32,2030,1696,,0,,8,0,24,0,no check,1,1,0,,,640052,,,1,4,1,,8,,2,0,,1,,,,0,0,0,0,0,0,,,,,0,0,,,,,99954,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
radicale,BACKEND,0,0,0,1,1000,8,2030,1696,0,0,,8,0,24,0,UP,1,1,0,,0,640052,0,,1,4,0,,8,,1,0,,1,,,,0,0,0,0,8,0,,,,8,0,0,0,0,0,0,99954,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
hugo,hugo,0,0,0,1,,8,271,424,,0,,2,0,6,0,no check,1,1,0,,,640052,,,1,5,1,,2,,2,0,,1,,,,0,0,0,0,0,0,,,,,0,0,,,,,183097,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
hugo,BACKEND,0,0,0,1,1000,2,271,424,0,0,,2,0,6,0,UP,1,1,0,,0,640052,0,,1,5,0,,2,,1,0,,1,,,,0,0,0,0,2,0,,,,2,0,0,0,0,0,0,183097,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
nginx,nginx-local,0,0,0,48,,37490,1114021941,258006337,,0,,0,0,0,0,UP,1,1,0,0,0,640052,0,,1,6,1,,37490,,2,0,,62,L7OK,200,0,0,34685,1050,1712,29,0,,,,,30,0,,,,,17,OK,,0,0,94,669,,,,Layer7 check passed,,2,3,4,,,,,,http,,,,,,,,
nginx,BACKEND,0,0,0,48,1000,37490,1114021941,258006337,0,0,,0,0,0,0,UP,1,1,0,,0,640052,0,,1,6,0,,37490,,1,0,,62,,,,0,34685,1050,1712,29,14,,,,37490,30,0,0,0,0,0,17,,,0,0,94,669,,,,,,,,,,,,,,http,,,,,,,,
chronograf,chronograf,0,0,0,0,,0,0,0,,0,,0,0,0,0,no check,1,1,0,,,640052,,,1,7,1,,0,,2,0,,0,,,,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
chronograf,BACKEND,0,0,0,0,1,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,0,640052,0,,1,7,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,0,0,0,0,0,0,0,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
minio,minio,0,0,0,0,,0,0,0,,0,,0,0,0,0,no check,1,1,0,,,640052,,,1,8,1,,0,,2,0,,0,,,,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
minio,BACKEND,0,0,0,0,1000,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,0,640052,0,,1,8,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,0,0,0,0,0,0,0,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
influxdb-admin,influxdb-admin,0,0,0,0,,0,0,0,,0,,0,0,0,0,no check,1,1,0,,,640052,,,1,9,1,,0,,2,0,,0,,,,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
influxdb-admin,BACKEND,0,0,0,0,1,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,0,640052,0,,1,9,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,0,0,0,0,0,0,0,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
influxdb-data,influxdb-data,0,0,0,0,,0,0,0,,0,,0,0,0,0,no check,1,1,0,,,640052,,,1,10,1,,0,,2,0,,0,,,,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
influxdb-data,BACKEND,0,0,0,0,1,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,0,640052,0,,1,10,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,0,0,0,0,0,0,0,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
torrent,torrent,0,0,0,0,,0,0,0,,0,,0,0,0,0,no check,1,1,0,,,640052,,,1,11,1,,0,,2,0,,0,,,,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
torrent,BACKEND,0,0,0,0,1,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,0,640052,0,,1,11,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,0,0,0,0,0,0,0,-1,,,0,0,0,0,,,,,,,,,,,,,,http,,,,,,,,
stats,BACKEND,0,0,1,1,1000,195,67998,8953906,0,0,,0,0,0,0,UP,0,0,0,,0,640052,,,1,12,0,,0,,1,1,,2,,,,0,194,0,0,0,0,,,,194,0,0,0,0,0,0,0,,,0,0,0,9,,,,,,,,,,,,,,http,,,,,,,,
