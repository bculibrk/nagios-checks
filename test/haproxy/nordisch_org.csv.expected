HAPROXY OK: 13 proxies found
chronograf BACKEND (act) UP	sess=0/1(0%) smax=0
http FRONTEND (act) OPEN	sess=2/10000(0%) smax=48
http sock-1 (act) OPEN	sess=0/10000(0%) smax=7
http sock-2 (act) OPEN	sess=0/10000(0%) smax=1
http sock-3 (act) OPEN	sess=2/10000(0%) smax=48
hugo BACKEND (act) UP	sess=0/1000(0%) smax=1
influxdb-admin BACKEND (act) UP	sess=0/1(0%) smax=0
influxdb-data BACKEND (act) UP	sess=0/1(0%) smax=0
minio BACKEND (act) UP	sess=0/1000(0%) smax=0
nginx BACKEND (act) UP	sess=0/1000(0%) smax=48
nginx nginx-local (act) UP	sess=0/-1(0%) smax=48
radicale BACKEND (act) UP	sess=0/1000(0%) smax=1
torrent BACKEND (act) UP	sess=0/1(0%) smax=0
